examples/hotrod/services/route/server.go

```diff
@@ -59,7 +59,7 @@ func (s *Server) Run() error {
 func (s *Server) createServeMux() http.Handler {
 	mux := tracing.NewServeMux(s.tracer)
 	mux.Handle("/route", http.HandlerFunc(s.route))
-	mux.Handle("/debug/vars", http.HandlerFunc(httpexpvar.Handler))
+	mux.Handle("/debug/vars", expvar.Handler()) // use exported handler
```

Comment:
note - this is not the only place the handler was used
```
CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/agent/agent-linux ./cmd/agent/main.go
pkg/metrics/builder.go:28:2: cannot find package "github.com/uber/jaeger/examples/hotrod/pkg/httpexpvar" in any of:
	/home/travis/gopath/src/github.com/uber/jaeger/vendor/github.com/uber/jaeger/examples/hotrod/pkg/httpexpvar (vendor tree)
	/home/travis/.gimme/versions/go1.9.linux.amd64/src/github.com/uber/jaeger/examples/hotrod/pkg/httpexpvar (from $GOROOT)
	/home/travis/gopath/src/github.com/uber/jaeger/examples/hotrod/pkg/httpexpvar (from $GOPATH)
```

Reply:
OK - my bad - I am on it.  👍

Reply:
OK - as far as I can tell that *is* the only place in the code.  However I forked `jaeger-tracing/jaeger' and this is still looking back at `uber/jaeger`.  I think I need to fork the repo from Uber and change it there and submit a PR.  Does that sound right to you Yuri?  If so you can just cancel this one.

Or, this can be a bigger PR where I change the imports over to `jaeger-tracing/jaeger' for hotrod...

Reply:
jaegertracing/jaeger is the new github location of uber/jaeger, the old location is redirected here. Your current PR is fine as is, you just need to fix another place:
```
./pkg/metrics/builder.go:28:	"github.com/uber/jaeger/examples/hotrod/pkg/httpexpvar"
```

Reply:
Hmm... this is likely an issue with the automated build tooling.  My change eliminates the need for that package (and thus the package is missing) in my PR.  That was kind of the point of my PR - that package was only needed prior to Go 1.8.

Reply:
yes, but your change doesn't touch the file `./pkg/metrics/builder.go` which still imports the old package. It needs to change similar to how this file is changed

Reply:
Ah!  Sorry I misunderstood.  I thought that package was only part of Hotrod!  Changes made and signed.  Tests passed.

Reply:
OK - commits squashed and all changes made.  Sorry - thought this would be a quick easy change.